plugins/wazo-yealink/v86/common.py

```diff
@@ -64,32 +72,32 @@ def _extract_from_ua(self, ua):
             if m:
                 raw_model, raw_version, raw_mac = m.groups()
                 try:
-                    mac = norm_mac(raw_mac.decode('ascii'))
+                    mac = norm_mac(six.ensure_str(raw_mac))
                 except ValueError as e:
                     logger.warning('Could not normalize MAC address "%s": %s', raw_mac, e)
                     return {
                         u'vendor': u'Yealink',
-                        u'model': raw_model.decode('ascii'),
-                        u'version': raw_version.decode('ascii'),
+                        u'model': six.ensure_str(raw_model),
+                        u'version': six.ensure_str(raw_version),
                     }
                 else:
                     return {
                         u'vendor': u'Yealink',
-                        u'model': raw_model.decode('ascii'),
-                        u'version': raw_version.decode('ascii'),
+                        u'model': six.ensure_str(raw_model),
+                        u'version': six.ensure_str(raw_version),
                         u'mac': mac,
                     }
         return None
 
     def _extract_from_path(self, request):
-        if request.path.startswith('/001565'):
-            raw_mac = path[1:-4]
+        if request.path[1:7] in KNOWN_MAC_PREFIXES:
```

Comment:
What happened with the other mac addresses in KNOWN_MAC_PREFIXES in the previous version?

Reply:
They only extracted the MAC from the request headers in that case. This will make it possible to have the MAC address another way in case they are not in the headers.